sql/analyzer/process.go

```diff
@@ -64,5 +55,11 @@ func trackProcess(ctx *sql.Context, a *Analyzer, n sql.Node) (sql.Node, error) {
 		return nil, err
 	}
 
-	return plan.NewQueryProcess(n, func() { processList.Done(pid) }), nil
+	// Don't wrap CreateIndex in a QueryProcess, as it is a CreateIndexProcess.
+	// CreateIndex will take care of marking the process as done on its own.
+	if _, ok := n.(*plan.CreateIndex); ok {
```

Comment:
why we cannot do the same as standard queries with create index queries? they are processing partitions too, and the process is measuring processed partitions.

Reply:
@erizocosmico ping

Reply:
This has nothing to do with measuring. If we wrap it with a QueryProcess the process will be marked as done as soon as the query returns, which is not when it has finished in the case of async indexes.